shell/platform/darwin/ios/FlutterTestsInfo.plist

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+  <key>CFBundleDevelopmentRegion</key>
+  <string>en</string>
+  <key>CFBundleExecutable</key>
+  <string>FlutterTests</string>
+  <key>CFBundleIdentifier</key>
+  <string>io.flutter.FlutterTests</string>
+  <key>CFBundleInfoDictionaryVersion</key>
+  <string>6.0</string>
+  <key>CFBundleName</key>
+  <string>FlutterTests</string>
+  <key>CFBundleVersion</key>
+  <string>1.0</string>
```

Comment:
do we need to worry about the fact that these version strings are hardcoded?

Reply:
For the case of witing a unit testing app, no.  For a real app, we'd probably want better handling of the plist in general.